src/pages/_document.tsx

```diff
@@ -5,6 +5,10 @@ class MyDocument extends Document {
     return (
       <Html lang="en">
         <Head>
+          <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap"
+          />
```

Comment:
@HonkingGoose  This is how you load fonts from Google Font now! Pretty simple since:

-  you only need to go to https://fonts.google.com, pick the desired font, and copy&paste the corresponding stylesheet link
- Next.js will apply corresponding optimizations
- [Next.js has its own ESLint rules](https://nextjs.org/docs/basic-features/eslint#eslint-plugin) so it will warn you if it's not set up properly

After the font is loaded, you need to set it up in Chakra's them within `customTheme.ts` file. [The default font family options in Chakra are "body", "heading" and "mono"](https://chakra-ui.com/docs/theming/theme#typography), but we can add more if necessary.

Reply:
>     * you only need to go to https://fonts.google.com, pick the desired font, and copy&paste the corresponding stylesheet link

Easy! 😄 



> * Next.js will apply corresponding optimizations
> 
> * [Next.js has its own ESLint rules](https://nextjs.org/docs/basic-features/eslint#eslint-plugin) so it will warn you if it's not set up properly

Even better, tell me when I'm stupid! 😄 

> After the font is loaded, you need to set it up in Chakra's them within `customTheme.ts` file. [The default font family options in Chakra are "body", "heading" and "mono"](https://chakra-ui.com/docs/theming/theme#typography), but we can add more if necessary.

Okay, will need to dive into this a bit more together when we're ready to use the Inter font. 😉